catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
I'm not sure that's the right approach. @stustison do you think the service should be changed in that way?